### PR TITLE
Performance optimization changes by VTEX Growth Hacking team

### PR DESCRIPTION
**Pull request by brunovaletta**
#### What problem is this solving?

Changes on the theme layout and blocks' properties to improve performance measured by Google Lighthouse. 

Previous performance score:
image

Score after changes:
image

#### How to test it?

A copy of this theme is installed here:
https://storetheme.vtex.com/?workspace=vtexprojectnewstoretheme

### CODE REVIEW

**Review by mateussaggin — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,8 +5,30 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+// VTEX Store Theme by Growth Hacking Team 
+
 ## [Unreleased]
 
+### Changed
+- Fixed height in `list-context.image-list#demo` example
```
Comment (victorhmp) [inline CHANGELOG.md:13]: This is a nitpick, but could you add `.`s to all entries? 😂

File: CHANGELOG.md [hunk 2]
```diff
@@ -5,8 +5,30 @@ All notable changes to this project will be documented in this file.
...
+- Pre-load prop in `list-context.image-list#demo`
+- image-rendering: auto and min-height: 450px in `sliderLayoutContainer--carousel` in vtex.slider-layout.css file
+- min-height in `.layoutContainer--shelf` and `slide--shelf` in vtex.slider-layout.css file
+- Height in `logo#mobile` and  `logo#desktop`
+
+### Removed
+- `__fold__.experimentalLazyAssets` from home page
+- Margin-bottom and padding-bottom from `flexRow--deals .flexCol` in vtex.flex-layout.css file
+- `flex-layout.row#2-desktop` block from header-layout.desktop
+- Removed `notification.bar#home` from header
```
Comment (victorhmp) [inline CHANGELOG.md:30]:
```suggestion
- `notification.bar#home` from header
```

**Review by victorhmp — CHANGES_REQUESTED** [review]
Just some minor suggestions ;)

Oh! And notice that a new major was released earlier today, so please rebase this branch with master, no conflicts should happen (after you undo the changes in `manifest.json`).

Some awesome results here!! Thanks so much 😄

**Review by victorhmp — CHANGES_REQUESTED** [review]

File: styles/css/vtex.flex-layout.css [hunk 3]
```diff
@@ -6,14 +6,15 @@
 @media screen and (min-width: 40em) {
   .flexRowContent--menu-link,
   .flexRowContent--main-header {
-    padding: 0 1rem;
-  }
+      padding: 0 1rem;
+    }
```
Comment (victorhmp) [inline styles/css/vtex.flex-layout.css:10]:
```suggestion
    padding: 0 1rem;
  }
```

File: styles/css/vtex.store-components.css [hunk 4]
```diff
@@ -67,25 +67,21 @@
 
 @media only screen and (min-width: 640px) {
   .newsletter .container {
-    padding: 140px;
-  }
-
+      padding: 140px;
+    }
```
Comment (victorhmp) [inline styles/css/vtex.store-components.css:71]:
```suggestion
    padding: 140px;
  }
```

File: styles/css/vtex.store-components.css [hunk 5]
```diff
@@ -67,25 +67,21 @@
...
   .label {
-    font-size: 40px;
-  }
-
+      font-size: 40px;
+    }
```
Comment (victorhmp) [inline styles/css/vtex.store-components.css:74]:
```suggestion
     font-size: 40px;
  }
```

File: styles/css/vtex.store-components.css [hunk 6]
```diff
@@ -67,25 +67,21 @@
...
   .infoCardContainer--info-card-home .infoCardTextContainer {
-    width: 30%;
-  }
-
+      width: 30%;
+    }
```
Comment (victorhmp) [inline styles/css/vtex.store-components.css:77]:
```suggestion
    width: 30%;
  }
```

File: styles/css/vtex.store-components.css [hunk 7]
```diff
@@ -67,25 +67,21 @@
...
-
+      width: 70%;
+    }
   .imageElement--storePrint {
-    margin-bottom: 0px;
-    max-height: 600px;
-  }
+      margin-bottom: 0px;
+      max-height: 600px;
+    }
```
Comment (victorhmp) [inline styles/css/vtex.store-components.css:84]:
```suggestion
    margin-bottom: 0px;
    max-height: 600px;
  }
```

**Review by victorhmp — CHANGES_REQUESTED** [review]

**Pull request merged**